packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -192,26 +207,29 @@ class ExportToCsvCommand extends Command {
             util.write(this, listOfUsers, fileName, 'organization details');
           } catch (error) {
             if (error.message) {
-              this.log(`Error: ${error.message}`);
+              this.log(`${util.formatError(error)}`);
```

Comment:
@abhinav-from-contentstack this string concatenation is not necessary. We can remove it here. this.log(util.formatError(error));

Reply:
Hi, I've removed the string concatenation.